hack/verify/lint.sh

```diff
@@ -30,10 +30,10 @@ LINTS=(
   deadcode errcheck gosimple govet ineffassign staticcheck \
   structcheck typecheck unused varcheck \
   # additional lints
-  golint gofmt misspell gochecknoinits unparam scopelint gosec
+  golint gofmt misspell gochecknoinits unparam scopelint
```

Comment:
should probably check if there are more new ones we should enable in a follow up